packages/libllvm/llvmgold.subpackage.sh

```diff
@@ -0,0 +1,4 @@
+TERMUX_SUBPKG_INCLUDE="
+lib/LLVMgold.so
+"
+TERMUX_SUBPKG_DESCRIPTION="LLVM gold plugin"
```

Comment:
Does this depend on clang or can gcc use it too? Add gold and whatever other dependencies are needed.

Reply:
Depends on `libllvm` (the parent package) and `libc++` (which is also a dependency of `libllvm`.) `llvmgold` itself does not hard-depend on other packages such as `binutils-gold`.

Reply:
Then rather I suppose `bunutils-gold` should depend on (or recommend) `llvmgold`.

Reply:
Yeah, recommending this sounds right, plus I think this should depend on the gold package, since it won't work without it. It's similar to how `gst-plugins-base` depends on `gstreamer`.

Reply:
But it should be noted that `gst-plugins-base` actually hard-depends on `gstreamer`, links against some shared libs in it. Not much similar.